### PR TITLE
Update PROJECT_LIFECYCLE.md

### PR DESCRIPTION
**Pull request by fredjn**
### Applicable Issues
N/A

### Description of the Change
Added short sentence regarding naming convention for new projects

### Alternate Designs
N/A

### Benefits
Make it clear that there is no strictly enforced project naming convention

### Possible Drawbacks
A risk projects with confusing name being added

### Sign-off
Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Fredrik Fristedt fredrik.fristedt@axis.com

### CODE REVIEW

File: PROJECT_LIFECYCLE.md [hunk 1]
```diff
@@ -38,6 +38,9 @@ their ability.
 - names of initial maintainers
 - infrastructure needs such as Git repositories
 
+Regarding project naming convenctions, the eiffel community enforces no name convention but meaningfull descriptive 
+names are encuraged.
```
Comment (t-persson) [inline PROJECT_LIFECYCLE.md:42]:
```suggestion
names are encouraged. 
```

File: PROJECT_LIFECYCLE.md [hunk 2]
```diff
@@ -38,6 +38,9 @@ their ability.
 - names of initial maintainers
 - infrastructure needs such as Git repositories
 
+Regarding project naming convenctions, the eiffel community enforces no name convention but meaningfull descriptive
```
Comment (t-persson) [inline PROJECT_LIFECYCLE.md:41]:
```suggestion
Regarding project naming convenctions, the eiffel community enforces no name convention but meaningful descriptive 
```

**Review by t-persson — CHANGES_REQUESTED** [review]
Spellcheck

File: PROJECT_LIFECYCLE.md [hunk 3]
```diff
@@ -38,6 +38,9 @@ their ability.
 - names of initial maintainers
 - infrastructure needs such as Git repositories
 
+Regarding project naming convenctions, the eiffel community enforces no name convention but meaningful descriptive
```
Comment (fdegir) [inline PROJECT_LIFECYCLE.md:41]:
```suggestion
Regarding project naming conventions, the Eiffel Community enforces no name convention but meaningful descriptive 
```

typo + capitals

Comment (fdegir) [reply PROJECT_LIFECYCLE.md:41]: as mentioned on Slack, I do not have a strong opinion about this so I vote 0/abstain from voting. ;)

**Review by m-linner-ericsson — COMMENTED** [review]
Good that we record this in writing. I don't have anything else to add except waiting for fix for Fatih comments

**Review by d-stahl-ericsson — APPROVED** [review]

**Review by fdegir — APPROVED** [review]

**Review by m-linner-ericsson — APPROVED** [review]

**Review by sodero — APPROVED** [review]

**Review by t-persson — APPROVED** [review]

**Review by henning-roos — APPROVED** [review]

**Review by e-backmark-ericsson — APPROVED** [review]

**Review by k-hallen-ericsson — APPROVED** [review]

**Comment by fdegir** [comment]
This amendment to project lifecycle passes so the PR can be merged. Thanks all for the reviews.

Please also take a look at the [ETOS project proposal](https://github.com/eiffel-community/community/issues/57) and comment/vote on it accordingly.

**Pull request merged**